docs/gateway_client.rst

```diff
@@ -1,80 +1,8 @@
 GatewayClient
 =============
-.. py:module:: starknet_py.net.gateway_client
-.. py:class:: GatewayClient
-
-.. automethod:: GatewayClient.__init__
-.. automethod:: GatewayClient.get_contract_addresses
-.. automethod:: GatewayClient.get_contract_addresses_sync
-.. automethod:: GatewayClient.call_contract
-.. automethod:: GatewayClient.call_contract_sync
-
-We recommend using :obj:`ContractFunction's call <starknet_py.contract.ContractFunction.call>` instead.
-
-.. automethod:: GatewayClient.get_block
-.. automethod:: GatewayClient.get_block_sync
-.. automethod:: GatewayClient.get_block_traces
-.. automethod:: GatewayClient.get_block_traces_sync
-
-.. automethod:: GatewayClient.get_code
-.. automethod:: GatewayClient.get_code_sync
-
-.. automethod:: GatewayClient.get_storage_at
-.. automethod:: GatewayClient.get_storage_at_sync
-.. automethod:: GatewayClient.get_transaction_status
-.. automethod:: GatewayClient.get_transaction_status_sync
-
-
-The possible statuses are:
-
-- **NOT_RECEIVED**: The transaction has not been received yet (i.e., not written to storage).
-- **RECEIVED**: The transaction was received by the operator.
-- **PENDING**: The transaction passed the validation and is waiting to be sent on-chain.
-- **REJECTED**: The transaction failed validation and thus was skipped.
-- **ACCEPTED_ON_L1**: The transaction was accepted on layer 1.
-- **ACCEPTED_ON_L2**: The transaction was accepted on layer 2.
```

Comment:
Is this documented somewhere?

Reply:
https://starknet.io/docs/hello_starknet/intro.html

Reply:
Okay, makes sense to me. We shouldn't be duplicating docs that exist for starknet already.

cc @war-in @Solpatium

Reply:
https://starknet.io/docs/hello_starknet/intro.html#interact-with-the-contract is the better url.

Reply:
I think @drknzz wasn't suggesting to include this link in our docs anyways.

Reply:
Ok, it is fine to remove it